cluster/charts/rook/values.yaml.tmpl

```diff
@@ -31,3 +31,8 @@ rbacEnable: true
 ## toleration. Choose between NoSchedule, PreferNoSchedule and NoExecute:
 # agent:
 #   toleration: NoSchedule
+
+## Override FlexVolume plugin directory
+## If unset, the Rook operator tries to discover it or uses a compiled in
+## default.
+#flexVolumeDirPath: /var/lib/kubelet/volumeplugins
```

Comment:
The default value here should probably match the default in kubernetes: ```/usr/libexec/kubernetes/kubelet-plugins/volume/exec/```

Reply:
@kokhang I've rebased to current master and changed the path in the comment.